ui/src/app/cohort-search/gender-chart/gender-chart.component.tsx

```diff
@@ -0,0 +1,128 @@
+import {Component, Input} from '@angular/core';
+import {getChartObj} from 'app/cohort-search/utils';
+import {ReactWrapperBase} from 'app/utils';
+import * as highCharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+import * as React from 'react';
+
+highCharts.setOptions({
+  lang: {
+    decimalPoint: '.',
+    thousandsSep: ','
+  },
+});
+
+interface Props {
+  data: any;
+}
+
+interface State {
+  options: any;
+}
+
+export class GenderChart extends React.Component<Props, State> {
+  readonly codeMap = {
+    'M': 'Male',
```

Comment:
Just a quick question, is this a good opportunity with the component conversion to move to using a more inclusive gender set for the chart?

Reply:
As far as the UI goes, we can do this anytime. I know it's been a re-identification issue in the past. @jscherdin any idea about this?

Reply:
We are just waiting on final CAPs bucket information before we change this